tests/e2e/sentinel/schedules/outbound.spec.js

```diff
@@ -126,105 +83,39 @@ describe('Outbound', () => {
+        chai.expect(inboxes.broken.length).to.equal(1);
 
-        chai.expect(inboxes.one).to.have.deep.members([
-          {
-            id: 'no_patient_nocd',
-            clinic: 'clinic_id',
-            fields: {
-              ab: 102,
-              cd: null,
-            },
-          },
-          {
-            id: 'all_fields_1',
-            clinic: 'other_clinic',
-            fields: {
-              ab: 120,
-              cd: 2,
-            },
-            patient: 'alpha',
-          },
-          {
-            id: 'all_fields_2',
-            clinic: 'some_clinic',
-            fields: {
-              ab: 0,
-              cd: -3,
-            },
-            patient: 'beta',
-          }
+        chai.expect(inboxes.working).to.have.deep.members([
+          {id: 'test-aaa'},
+          {id: 'test-zzz'}
         ]);
 
-        chai.expect(inboxes.two).to.have.deep.members([
-          {
-            keys: ['keya', 'keyb', 'keyc', 'keyd', 'keypatient_id'],
-            values: ['value5', 'value4', 'value10', 'value8', 'valuealpha']
-          },
-          {
-            keys: ['keya', 'keyb', 'keyc', 'keyd', 'keypatient_id'],
-            values: ['value-5', 'value20', 'value9', 'value12', 'valuebeta']
-          },
-          {
-            keys: ['keya', 'keyb', 'keyc', 'keyd', 'keye', 'keyf'],
-            values: ['value1', 'value2', 'value3', 'value4', 'value5', 'value6']
-          }
+        chai.expect(inboxes.broken).to.have.deep.members([
+          {id: 'test-aaa'}
         ]);
       })
-      .then(() => utils.getDocs(docs.map(doc => doc._id)))
-      .then(results => {
-        // original docs were not updated
-        results.forEach(doc => chai.expect(doc._rev.startsWith('1-')).to.equal(true));
-      })
       .then(() => utils.sentinelDb.allDocs({ keys: tasks.map(task => task._id), include_docs: true }))
       .then(result => {
-        // all sent task:outbound docs have been deleted, all unsent still exist unchanged
-        const sentTasks = [
-          'task:outbound:no_patient_nocd',
-          'task:outbound:all_fields_1',
-          'task:outbound:all_fields_2',
-          'task:outbound:for_two',
-          'task:outbound:to_delete'
-        ];
-        result.rows.forEach(task => {
-          if (sentTasks.includes(task.id)) {
-            chai.expect(task.doc).to.equal(null);
-          } else {
-            chai.expect(task.doc).not.to.equal(null);
-            chai.expect(task.doc).to.deep.include(tasks.find(t => t._id === task.id));
-          }
+        chai.expect(result.rows.length).to.equal(2);
+        chai.expect(result.rows[0].doc).to.deep.equal({
+          _id: `task:outbound:test-aaa`,
+          _rev: result.rows[0].doc._rev,
+          type: 'task:outbound',
+          doc_id: 'test-aaa',
+          queue: ['broken'],
         });
-      })
-      .then(() => sentinelUtils.getInfoDocs(docs.concat(docsToDelete).map(doc => doc._id)))
-      .then(infos => {
```

Comment:
I've noticed that this test was flaking in this section, where checking the info docs, which you completely removed. 
I couldn't determine whether it's a flake or a real failure. Can we please keep this? I've looked over your other e2e test and it doesn't cover checking info docs either.

Reply:
Doh! I thought my e2e tests did this, but thinking about it both sets should, as they are both responsible for it. Let me add that!